lib/src/models/subreddit.dart

```diff
@@ -423,14 +420,80 @@ class Subreddit extends SubredditRef with RedditBaseInitializedMixin {
   }
 }
 
-// TODO(bkonyi): implement.
-// Provides functions to interact with the special [Subreddit]'s filters.
-/*class SubredditFilters {
-  final Subreddit _subreddit;
-  SubredditFilters(this._subreddit) {
-    throw new DRAWUnimplementedError();
+/// Provides functions to interact with the special [Subreddit]'s filters.
+class SubredditFilters {
+  static final RegExp _userRegExp = new RegExp(r'{user}');
```

Comment:
Should we move all instances of `RegExp`, to a single static Utils Class.

The rationale for the comment isn't a strong one though; I can't imagine the api changing these.

Reply:
It might be worth moving into `api_paths.dart` at some point, but I'll do this later.